packages/react-router/src/server/wrapSentryHandleRequest.ts

```diff
@@ -47,6 +49,33 @@ export function sentryHandleRequest(originalHandle: OriginalHandleRequest): Orig
         });
       }
     }
+
     return originalHandle(request, responseStatusCode, responseHeaders, routerContext, loadContext);
   };
 }
+
+/** @deprecated Use `wrapSentryHandleRequest` instead. */
+export const sentryHandleRequest = wrapSentryHandleRequest;
+
+/**
+ * Injects Sentry trace meta tags into the HTML response by piping through a transform stream.
+ * This enables distributed tracing by adding trace context to the HTML document head.
+ *
+ * @param body - PassThrough stream containing the HTML response body to modify
+ */
+export function getMetaTagTransformer(body: PassThrough): Transform {
+  const headClosingTag = '</head>';
+  const htmlMetaTagTransformer = new Transform({
```

Comment:
l: As discussed I'm not 100% sure if `Transform` is supported in WinterTC runtimes (edge, CloudFlare) but we can cross this bridge when we get to it.